docs/git-commands.md

```diff
@@ -0,0 +1,94 @@
+# Git Hub Command Guide
+The following is number of scenarios that you will encounter while contributing to this repo, a convention in this document is to use `<< >>` to indicate text that you will need to replace when running the commands on your own machine.
+
+## Clone this REPO
+Before you begin, you will need to clone this repo
+```
+git clone https://github.com/MountSionCBSSecondary/SchoolWebsite.git
+```
+The above command will download a copy of the repo, this clone or copy will have the remote name of `origin`.
```

Comment:
Sentence reads funny here. A full stop after repo would I think help the flow of the sentence.

Reply:
I use the following as it follows git protocol: ie rename master to upstream and setting this to nopush

# 1. clone a master repo
git clone MAIN_GIT_SSH_URL
git remote -v

# 2. rename origin to upstream
git remote rename origin upstream

# 3. fork repo from github and add local fork (origin)
git remote add origin FORK_SSH_URL
git remote -v

# 4. set no push on upstream master
git remote set-url --push upstream no_push

# 5. Make sure you're up-to-date with pull and push
git pull upstream master
git push origin master